OtripleS.Web.Api.Tests.Unit/Services/UserServiceTests/UserServiceTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using Moq;
+using OtripleS.Web.Api.Brokers.DateTimes;
+using OtripleS.Web.Api.Brokers.Loggings;
+using OtripleS.Web.Api.Brokers.Storage;
+using OtripleS.Web.Api.Models.Users;
+using OtripleS.Web.Api.Services.Users;
+using Tynamix.ObjectFiller;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.UserServiceTests
+{
+    public partial class UserServiceTests
+    {
+        private readonly Mock<IStorageBroker> storageBrokerMock;
+        private readonly Mock<ILoggingBroker> loggingBrokerMock;
+        private readonly Mock<IDateTimeBroker> dateTimeBrokerMock;
+        private readonly IUserService userService;
+
+        public UserServiceTests()
+        {
+            this.storageBrokerMock = new Mock<IStorageBroker>();
+            this.loggingBrokerMock = new Mock<ILoggingBroker>();
+            this.dateTimeBrokerMock = new Mock<IDateTimeBroker>();
+
+            this.userService = new UserService(
+                storageBroker: this.storageBrokerMock.Object,
+                loggingBroker: this.loggingBrokerMock.Object,
+                dateTimeBroker: this.dateTimeBrokerMock.Object);
+        }
+
+        //private User CreateRandomUser() =>
+        //    CreateUserFiller(dates: DateTimeOffset.UtcNow).Create();
+
+        //private static Filler<User> CreateUserFiller(DateTimeOffset dates)
+        //{
+        //    var filler = new Filler<User>();
+
+        //    filler.Setup()
+        //        .OnType<DateTimeOffset>().Use(dates);
+
+        //    return filler;
+        //}
+
+        private User CreateRandomUser()
+        {
+            var user = new User
+            {
+                Id = new Guid(),
+                UserName = new Filler<EmailAddresses>().Create().ToString(),
+                Name = new Filler<RealNames>().Create().ToString(),
+                FamilyName = new Filler<RealNames>().Create().ToString(),
+                CreatedDate = DateTimeOffset.UtcNow,
+                UpdatedDate = DateTimeOffset.UtcNow
+
+            };
+
+            return user;
+        }
```

Comment:
@hassanhabib `Filler<User>` had problems so I tried to create a user this way. Should I continue like this?